packages/website/src/server/index.ts

```diff
@@ -28,17 +47,20 @@ function getServerUrl(server: http.Server) {
 }
 
 function getWebsiteWebpackConfig(
-  options: {
-    projectPath: string;
+  {
```

Comment:
This is a bit weird, do you need to give it a type? can you maybe just create a type alias for readability?

Reply:
Could you clarify what you mean?

Reply:
I mean that instead of giving the type inline create an interface for it:
`interface bla { outputPath: string;  production: boolean;  projectOptions: IProjectOptions; }`
and then use it as the type for getWebsiteWebpackConfig and getMetadataWebpackConfig